tests/clusters/test_cluster.py

```diff
@@ -142,6 +142,10 @@ def test_cache_connections() -> None:
         "localhost", 8000, "default", "", "default", 8001, {"events"}, True
     )
 
+    cluster_2 = cluster.ClickhouseCluster(
+        "localhost", 8000, "default", "", "default", 8001, {"transsactions"}, True
```

Comment:
not sure if it matters really but `transsactions` -> `transactions`

Reply:
updated, thanks